### DISCUSSION TITLE
TV Vest Vestland

### DISCUSSION THREAD

**Discussion by ghost** [iptv-org/community#352] [Show and tell]
### Channel Name

TV Vest Vestland

### Channel ID

TVVestVestland.no

### Website

https://video.tvvest.no/se-tv-vest-vestland-direkte

### Stream URL

https://live01.delivery.twentythree.com/live/amlst:iel8oipvbtsc6tbt8vttjuk3cc:20918017-d27c6c32d3789bc7c10d.mp4/chunklist_b2128000.m3u8

### Please confirm the following

- [X] I have read [Contributing Guide](https://github.com/iptv-org/iptv/blob/master/CONTRIBUTING.md#request-a-channel)

**Comment by freearhey** [comment]
https://live01.delivery.twentythree.com/live/amlst:iel8oipvbtsc6tbt8vttjuk3cc:20918017-d27c6c32d3789bc7c10d.mp4/chunklist_b2128000.m3u8 (offline)
<img width="836" alt="image" src="https://user-images.githubusercontent.com/7253922/232229371-bb8af174-99ca-4b1b-ae88-4723b36f2566.png">
